FILE packages/desktop/build/entitlements.mac.inherit.plist
RANGE {"path":"packages/desktop/build/entitlements.mac.inherit.plist","old_lines":[7,10],"new_lines":[7,12],"diff_block_lines":[5,11],"hunk_header":"@@ -7,4 +7,6 @@","substantive_changed_lines":2}
@@ -7,4 +7,6 @@
   <key>com.apple.security.cs.allow-unsigned-executable-memory</key>
   <true/>
+  <key>com.apple.security.device.audio-input</key>
+  <true/>
 </dict>
 </plist>
